src/rrt/BiRRT.hpp

```diff
@@ -202,7 +202,7 @@ class BiRRT {
     int _minIterations;
 
     int _solutionLength;
-    const Node<T>* _startSolutionNode, *_goalSolutionNode;
+    const Node<T>*_startSolutionNode, *_goalSolutionNode;
```

Comment:
Missing a space

Reply:
Stylize seems to be _very_ insistent that this is the correct formatting. I'll change it, but we'll just have to ignore it here.